CHANGELOG.md

```diff
@@ -13,6 +13,9 @@
 - Create Kotlin extension for saving Optional preferences
 - Modify `PrescribedDrug` to include `teleconsultation_id`
 - Change `BloodPressureEntryEffectHandler` create and update measurement flows to be mostly synchronous
+- Add custom traces in Firebase Performance Monitoring for the following flows:
+  - Record new blood pressure measurement
+  - Update existing blood pressure measurement
```

Comment:
`bpEntry_markAppointmentsVisited` would be considered in updating blood pressure measurement record, right?